src/librustdoc/html/render/context.rs

```diff
@@ -250,6 +250,8 @@ impl<'tcx> Context<'tcx> {
     fn build_sidebar_items(&self, m: &clean::Module) -> BTreeMap<String, Vec<NameDoc>> {
         // BTreeMap instead of HashMap to get a sorted output
         let mut map: BTreeMap<_, Vec<_>> = BTreeMap::new();
+        let mut inserted: FxHashMap<ItemType, FxHashSet<Symbol>> = FxHashMap::default();
```

Comment:
It's possible because of reexports. Initially, I had rewrote using a `BTreeMap` but then I discovered that we had an option to keep the declaration order. So I had to revert that and instead using a `Hashmap` alongside...